CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # App Center SDK for Unity Editor Extensions Change Log
 
-## Release 2.0.1 (in development)
+## Version 2.1.0 (Under development)
+
+### App Center Push
```

Comment:
Maybe this change should be marked as 'Breaking change' as it was done for 'Auth' module in previous major version?